bundles/org.eclipse.equinox.simpleconfigurator/src/org/eclipse/equinox/internal/simpleconfigurator/ConfigApplier.java

```diff
@@ -19,6 +19,7 @@
 import java.net.*;
 import java.util.*;
 import java.util.concurrent.CountDownLatch;
+import java.util.stream.Collectors;
 import org.eclipse.equinox.internal.simpleconfigurator.utils.*;
 import org.osgi.framework.*;
 import org.osgi.framework.namespace.*;
```

Comment:
Why is PackageAdmin still imported?

Reply:
It still uses packagage admin on other places, but I wanted to start with one small thing here.